modules/default/weather/providers/openmeteo.js

```diff
@@ -0,0 +1,549 @@
+/* global WeatherProvider, WeatherObject */
+
+/* MagicMirror²
+ * Module: Weather
+ * Provider: Open-Meteo
+ *
+ * By Andrés Vanegas
+ * MIT Licensed
+ *
+ * This class is a provider for Open-Meteo, based on Andrew Pometti's class
+ * for Weatherbit.
+ */
+// https://www.bigdatacloud.com/docs/api/free-reverse-geocode-to-city-api
+const GEOCODE_BASE = "https://api.bigdatacloud.net/data/reverse-geocode-client";
+
+WeatherProvider.register("openmeteo", {
+	// Set the name of the provider.
+	// Not strictly required, but helps for debugging.
+	providerName: "Open-Meteo",
+
+	// Set the default config properties that is specific to this provider
+	defaults: {
+		apiBase: "https://api.open-meteo.com/v1",
+		lat: 0,
+		lon: 0,
+		past_days: 0,
+		type: "current",
+		timezone: null
+	},
+
+	units: {
```

Comment:
We recentyl changed how the weatherproviders handle units:
Please use internally ony metric units (celsius for temperature, meters per second for wind). So you have to make sure that either a) the api returns metric units or b) cif it returns imperial, convert them when you get them

The weather module will then convert the units from the weatherpovider once they are about to be displayed into what the user has configured. So nothing for the provider to do here

Reply:
To be clear:
- Units will be "stored" in weather object only in metric values for all providers?